assets/model_monitoring/components/src/data_quality_statistics/compute_data_quality_statistics.py

```diff
@@ -111,65 +111,77 @@ def get_unique_value_list(df: pyspark.sql.DataFrame) -> ps.DataFrame:
         schema=metadata_schema,
     )
 
-    unique_vals_df = unique_vals_df.to_pandas_on_spark()
-
     return unique_vals_df
 
 
-def compute_min_df(df: ps.DataFrame) -> ps.DataFrame:
-    """
-    Compute the minimum value for each feature in the input DataFrame.
-
-    Args:
-        df: A pandas-on-Spark DataFrame.
-
-    Returns:
-        A pandas-on-Spark DataFrame containing two columns: "featureName" and "min_value".
-        The "featureName" column lists the name of each feature in the input DataFrame, and
-        the "min_value" column lists the minimum value for each feature.
-    """
-    min_vals = ps.DataFrame(data=df.min(axis=0)).reset_index()
-    min_vals.columns = ["featureName", "min_value"]
-
-    return min_vals
-
-
-def compute_max_df(df: ps.DataFrame) -> ps.DataFrame:
+def compute_max_and_min_df(df: pyspark.sql.DataFrame, dtype_df: pyspark.sql.DataFrame) -> pyspark.sql.DataFrame:
     """
     Compute the maximum value for each feature in the input DataFrame.
 
     Args:
-        df: A pandas-on-Spark DataFrame.
+        df: A spark data frame with features to calculate max value and min value。
+        dtype_df: A spark data frame for datatype for each featureName
 
     Returns:
-        A pandas-on-Spark DataFrame containing two columns: "featureName" and "max_value".
+        A Spark DataFrame containing two columns: "featureName" and "max_value".
```

Comment:
This comment should be changed to "max_value" and "min_value"

Reply:
Also, line 128

Reply:
fixed the docstring